classes/PartyCohort.php

```diff
@@ -448,9 +449,12 @@ public static function calculatePositions($quiet=true){
 
             $cohort_count++;
 
+            $db->query("START TRANSACTION");
```

Comment:
And in PHP with PDO that we use, you call $db->conn->beginTransaction() and $db->conn->commit();